cloud/deploys/dev_azure/README.md

```diff
@@ -3,38 +3,62 @@ This is a deploy directory that allows individuals to create a deploy of the
 app. We intentionally don't use a shared state with this terraform directory
 to allow for everyone to have their own app. 
 
-The process to deploy is similar
-to the staging process but we do not have to do the shared state set up. It 
-should be `terraform init` and then `terraform apply`
+# Running the dev deploy
+## Copy the civiform_config.example.sh 
+Copy the civiform_config.example.sh into civiform_config.sh and
+ change the required variables. 
 
-# Local Docker Build to Remote Azure Deploy
-If you want to do local onto terraform we build/tag/deploy the docker image 
-and then update the azure app service to point to the local image. 
+## Get access to correct technologies
+You will need to reach out to team members to get accounts for the following:
+- Azure
+- AWS
+- Login Radius Civiform-Staging 
 
-## 1. Build, Tag and Push the Docker Image
-This should take like 30 minutes (the push takes the longest).
+## Setup Login Radius For Local Development
+Go to [Login Radius Dashboard](https://dashboard.loginradius.com/) and click
+configure a civiform integration. Choose the outbound SSO Saml.
+
+From there add an app with Sp initiated login and pick a name (this gets put
```

Comment:
How exactly do you do this?  If you create a new app, there's many fields to fill out, and there's no "copy an existing app" that I see.